<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="CSSBackgrounds.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Luis M Bolaños</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="ResInt.html">Research</a>
</li>
<li>
  <a href="Fieldwork.html">Fieldwork</a>
</li>
<li>
  <a href="Publications.html">Publications</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="http://github.com/lbolanos68">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/lbolanos68">
    <span class="fab fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/luis-bolanos-4b703b43/">
    <span class="fab fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<h1>
Publications
</h1>
<p>A list of my publications is also available on <a href="https://scholar.google.com/citations?user=okVkAH8AAAAJ&amp;hl=en&amp;authuser=1"><strong>Google scholar profile</strong></a></p>
<right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
14 Washburn, Q., Wolf, S.,Well, J., Noell, S., Lee, CP., Bolaños, L.M., Giovannoni, S.J., Suffridge, C. P. (2021) Oligotrophic A marine microbiology board game-based activity for high school science classrooms. The Science Teacher, 88(5). <a href="https://www.nsta.org/science-teacher/science-teacher-mayjune-2021/oligotrophic"><strong>https://www.nsta.org/science-teacher/science-teacher-mayjune-2021/oligotrophic</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
13 Bolaños, L.M., Choi, C.J., Worden, A.Z., Baetge, N., Carlson, C.A., Giovannoni, S.J. (2021) Seasonality of the microbial community composition in the North Atlantic. Frontiers in Marine Science, 8:624164. <a href="https://doi.org/10.3389/fmars.2021.624164"><strong>doi:10.3389/fmars.2021.624164</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
12 Buchholz, H.H., Michelsen, M., Bolaños, L.M., Browne, E., Allen, M.J., Temperton, B. (2021) Efficient Dilution-to-Extinction isolation of novel virus-host model systems for fastidious heterotrophic bacteria. The ISME Journal, 1-14. <a href="https://doi.org/10.1038/s41396-020-00872-z"><strong>doi.org/10.1038/s41396-020-00872-z</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
11 Suffridge, C. P., Bolaños L.M., Worden, A.Z., Bergauer, K., Morré J., Behrenfeld, M.J., and Giovannoni, S. J. (2020) Exploring Vitamin B1 cycling and its connections to the microbial community in the North Atlantic Ocean. Frontiers in Marine Science, 7:606342. <a href="https://doi.org/10.3389/fmars.2020.606342"><strong>doi.org/10.3389/fmars.2020.606342</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
10 Maas, A.E., Liu, S., Bolaños, L.M., Widner, B., Parsons, R., Kujawinski, E.B., Blanco-Bercial, L., Carlson, C.A. (2020) Migratory Zooplankton Excreta and its Influence on Prokaryotic Communities. Frontiers in Marine Science. 7:573268. <a href="https://doi.org/10.3389/fmars.2020.573268"><strong>doi.org/10.3389/fmars.2020.573268</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
9 Liu, S., Baetge, N., Comstock, J., Opalk, K., Parsons, R., Halewood, E., English, C., Giovannoni, S.J., Bolaños, L.M., Nelson, C.E.,Vergin, K., Carlson, C.A. (2020). Stable isotope probing identifies bacterioplankton lineages capable of utilizing dissolved organic matter across a range of bioavailability. Frontiers in Microbiology. 11:2364. <a href="https://doi.org/10.3389/fmicb.2020.580397"><strong>doi.org/10.3389/fmicb.2020.580397</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
8 Bolaños, L.M., Karp-Boss, L., Choi, C.J., Worden, A.Z., Graff, J.R., Haëntjens, N., Chase, A.P., Della Penna, A., Gaube, P., Morison, F., Menden-Deuer, S., Westberry, T.K., O’Malley, R.T., Boss, E., Behrenfeld, M.J., Giovannoni, S.J. (2020). Small phytoplankton dominate western North Atlantic biomass. The ISME Journal. 14:1663–1674. <a href="https://doi.org/10.1038/s41396-020-0636-0"><strong>doi.org/10.1038/s41396-020-0636-0</strong></a>
</p>
</right> Featured on <a href="https://www.theguardian.com/environment/2020/apr/03/oceans-capacity-to-absorb-co2-overestimated-study-suggests"><strong>The Guardian (3 Apr 2020)</strong></a> North Atlantic's capacity to absorb CO2 overestimated, study suggests by Jonathan Watts <br> Featured on <a href="https://www.molecularecologist.com/2020/04/a-bloom-by-any-other-name/"><strong>The Molecular Ecologist (7 Apr 2020)</strong></a> A bloom by any other name by Kelle Freel <br> Featured on <a href="https://www.nature.com/collections/acidjaejdg"><strong>The ISME Journal 2020 Best Paper Award (7 Apr 2020)</strong></a> Shortlisted among the most impactful articles in the journal <br> Featured on <a href="https://www.corvallisadvocate.com/2020/oregon-states-year-in-science/"><strong>The Corvallis Advocate: Oregon State’s Year in Science (Dec 26 2020)</strong></a> <br> Press release <a href="https://today.oregonstate.edu/news/smaller-expected-phytoplankton-may-mean-less-carbon-sequestered-sea-bottom"><strong>Newsroom Oregon State University</strong></a> <br> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
7 Liu, S., Parsons, R., Opalk, K., Baetge, N., Giovannoni, S., Bolaños, L.M., Kujawinski, E.B., Longnecker, K., Lu, Y., Halewood, E., Carlson, C.A. (2020). Different carboxyl-rich alicyclic molecules (CRAM) proxy compounds select distinct bacterioplankton for oxidation of DOM in the mesopelagic Sargasso Sea. Limnology and Oceanography. 65: 1532–1553. <a href="https://doi.org/10.1002/lno.11405"><strong>doi.org/10.1002/lno.11405</strong></a>
</p>
</right> Featured on <a href="https://www.eurekalert.org/pub_releases/2020-02/uoc--bds022620.php"><strong>Eurekalert.org (26 Feb 2020)</strong></a> Breaking down stubborn molecules in the ocean <br> Featured on <a href="https://www.sciencedaily.com/releases/2020/02/200226130521.htm"><strong>Sciencedaily (26 Feb 2020)</strong></a> Breaking down stubborn molecules in the ocean <br> Press release <a href="https://www.news.ucsb.edu/2020/019802/breaking-down-stubborn-molecules"><strong>The Current University of California Santa Barbara</strong></a> <br> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
6 Behrenfeld, M., Moore, R., Hostetler, C., Graff, J., Gaube, P., Russell, L., Gao Chen, G., Doney, C., Giovannoni, S., Liu, H., Proctor, C., Bolaños, L.M., Baetge, N., Davie-Martin, C., Westberry, T., Bates,T., Bell, T., Bidle, K., Boss, E., Brooks, S., Cairns, B., Carlson, C., Halsey, K., Harvey, E., Hu, C., Karp-Boss, L., Kleb, M., Menden-Deuer, S., Quinn, P., Scarino, A., Anderson, B., Chowdhary, J., Crosbie, E., Ferrare, R., Hair, J., Hu, Y., Janz, S., Redemann,J., Saltzman, E., Shook, M., Siegel, D., and Ziemba L. (2019). The North Atlantic Aerosol and Marine Ecosystem Study (NAAMES): Science motive and mission overview. Frontiers in Marine Science, 6:122. <a href="https://doi.org/10.3389/fmars.2019.00122"><strong>doi.org/10.3389/fmars.2019.00122</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
5 Bolaños, L.M., Rosenblueth, M., Manrique de Lara, A., Migueles-Lozano, A., Gil-Aguillón, C., Mateo-Estrada, V., González-Serrano, F., Santibáñez-López, C., García-Santibáñez, T., and Martínez-Romero. E. (2019). Cophylogenetic analysis suggests cospeciation between the Scorpion Mycoplasma Clade symbionts and their hosts. PLoS ONE 14: e0209588. <a href="https://doi.org/10.1371/journal.pone.0209588"><strong>doi.org/10.1371/journal.pone.0209588</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
4 Bolaños, L.M.,Rosenblueth, M., Castillo‐Ramírez, S., Figuier‐Huttin, G., and Martínez‐Romero, E. (2016). Species‐specific diversity of novel bacterial lineages and differential abundance of predicted pathways for toxic compound degradation in scorpion gut microbiota. Environmental microbiology, 18:1364-1378. <a href="https://doi.org/10.1111/1462-2920.12939"><strong>doi.org/10.1111/1462-2920.12939</strong></a>
</p>
</right> Featured on <a href="https://microbiomedigest.com/2015/06/11/june-11-2015/"><strong>Microbiome Digest – Bik's Picks (11 Jun 2015)</strong></a> Animal microbiome:Species-specific diversity of novel bacterial lineages and differential abundance of predicted pathways for toxic compound degradation in scorpion gut microbiota. <br> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
3 Bolaños, L.M., Servín-Garcidueñas, L.E., and Martínez-Romero, E. (2015). Arthropod–Spiroplasma relationship in the genomic era. FEMS microbiology ecology, 91:1-8. <a href="https://doi.org/10.1093/femsec/fiu008"><strong>doi.org/10.1093/femsec/fiu008</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
2 Degli Esposti, M., Rosas-Pérez, T., Servín-Garcidueñas, L.E., Bolaños, L.M., Rosenblueth, M., and Martínez-Romero, E. (2015). Molecular evolution of cytochrome bd oxidases across proteobacterial genomes. Genome biology and evolution, 7:801-820.<a href="https://doi.org/10.1093/gbe/evv032"><strong>doi.org/10.1093/gbe/evv032</strong></a>
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
1 Ramírez-Puebla, S.T., Servín-Garcidueñas, L.E., Jiménez-Marín, B., Bolaños, L.M., Rosenblueth, M., Martínez, J., Rogel, M.A., Ormeño-Orrillo, E., and Martínez-Romero, E. (2013). Gut and root microbiota commonalities. Applied and environmental microbiology, 79:2-9.<a href="https://doi.org/10.1128/AEM.02553-12"><strong>doi.org/10.1128/AEM.02553-12</strong></a>
</p>
</right> <br> <br>
<h2>
Contributions to symposia
</h2>
<h3>
<strong>Ocean Sciences meeting. San Diego, CA. 2020</strong>
</h3>
<right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
8 ME11A-01 Linking phytoplankton diversity in the North Atlantic to the annual cycle of ecological disturbance and recovery. L Karp-Boss, L M Bolaños, S J Giovannoni, J Gra􏰂, C Carbonell-Moore, N Haentjens, A Chase, E Boss, M Behrenfeld
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
7 MM12A-04 Protist plankton seasonality and community structure in the epi- and mesopelagic at BATS. L Blanco Bercial, R J Parsons, L M Bolaños, N McDonald, S J Giovannoni, R Curry
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
6 ED34A-3641 Pntroducing Oligotrophic, a Board Game Designed as an Interactive Learning Experience in Microbial Ecology and Biogeochemistry, a Tool for Science Communication, and Curriculum for Elementary through High School Teachers. C Su􏰂ridge, Q L Washburn, J Well, S Noell, S S Wolf, C P Lee, L M Bolaños, S J Giovannoni
</p>
</right> <br> <right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
5 OB43C-05 Exploring Vitamin B1 Cycling and its Connections to the Microbial Community in the North Atlantic Ocean. C Su􏰂ridge, L M Bolaños, A Z Worden, K Bergauer, J Morré, M Behrenfeld, S J Giovannoni
</p>
<p></right></p>
<h3>
<strong>Aquatic Sciences meeting. San Juan, PR. 2019</strong>
</h3>
<right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
4 SS011 Diverse Phytoplankton Form the western North Atlantic Spring Bloom. , L M Bolaños, S Menden-Deuer, L Karp-Boss, F Morison, C J Choi, A Z Worden, J Graff, C Carbonell-Moore, T K Westberry, A P Chase, N Haëntjens, P Gaube, E Boss, M Behrenfeld, S J Giovannoni
</p>
<p></right></p>
<right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
3 CS013 Stable isotope probing identifies bacterioplankton lineages capable of utilizing DOM across a range of bioavailability. S Liu, N Huynh, J Comstock, K Opalk, E Halewood, C English, R Parsons, S Giovannoni, L M Bolaños, C Carlson
</p>
<p></right></p>
<h3>
<strong>Ocean Sciences meeting. Portland, OR. 2018</strong>
</h3>
<right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
2 MM24B NAAMES (North Atlantic Aerosols and Marine Ecosystems Study) observations of non-diatom dominated plankton communities in late autumn and spring. L M Bolaños, S Menden-Deuer, L Karp-Boss, F Morison, E Boss, A P Chase, N Haëntjens, C J Choi, A Z Worden, J Graff,T K Westberry, M Behrenfeld, K Halsey, C Davie-Martin, C A Carlson, S J Giovannoni
</p>
<p></right></p>
<h3>
<strong>8th International Symbiosis Society (ISS) congress. Lisbon, Portugal. 2015</strong>
</h3>
<right>
<p style="font-family:Geneva;color:black;text-align:justify;font-size:16px;">
1 IFM Species-specific diversity of novel bacterial lineages and differential abundance of pathways for toxic compound degradation in scorpion gut microbiota. L M Bolaños, M Rosenblueth, S Castillo-Ramirez, E Martínez-Romero
</p>
<p></right></p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
